Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -360,12 +361,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $showProgress = $resolver->getProgress();
 
         if ($showProgress) {
-            $fileProcessedEventListener = function (FixerFileProcessedEvent $event) use ($output) {
-                $output->write($event->getStatusAsString());
-            };
-
             $this->fixer->setEventDispatcher($this->eventDispatcher);
-            $this->eventDispatcher->addListener(FixerFileProcessedEvent::NAME, $fileProcessedEventListener);
+            $stdErr = fopen('php://stderr', 'w');
```

Comment:
$stdErr is needed only for `ProcessOutput`. Shouldn't it handle opening and closing stderr by itself ?